### ISSUE TITLE
Plugins: should be able to load in datastacks, json, logfiles

### ISSUE THREAD

**Issue by dcdenu4** [natcap/invest#1700]
Currently this is failing because InVEST looks for an invest-like model in it's given environment space (see [models.py line 27](https://github.com/natcap/invest/blob/e6318a62e312eb6194ba574d7f97034a9844f48a/src/natcap/invest/models.py#L27)). But, the plugin is installed in the natcap namespace in it's isolated mamba env. 

I feel like this was partly redesigned specifically for plugins, but the separate environment issue seems to be a big hurdle to this being generalizable.

The specific error I see when trying is:
```
  File "C:\Users\ddenu\Workspace\Repositories\venvs\schisto\Lib\site-packages\natcap\invest\ui_server.py", line 168, in post_datastack_file
    model_id = PYNAME_TO_MODEL_ID_MAP[stack_info.model_name]
               ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^
KeyError: 'natcap.invest.schistosomiasis'
```

**Comment by emlys** [comment]
Yes, I think `server_requests.fetchDatastackFromFile` should call the plugin server, then this should work without changes to invest.

**Comment by emlys** [comment]
Done in #1759
